### PR TITLE
Some changes

### PR DESCRIPTION
**Pull request by ahoulgrave**
I like your program as it's super simple, i just refactored a bit the code so it's clearer, hope it's ok for you. Please do review and comment.

Thanks for sharing

### CODE REVIEW

**Comment by ahoulgrave** [comment]
Will add a Dockerfile

**Comment by klipitkas** [comment]
Hey, thanks a lot for the contribution. There are some merge conflicts however that need to be resolved.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM golang:1.13-alpine as base
```
Comment (klipitkas) [inline Dockerfile:1]: We could use `scratch` as the base image in order to reduce the docker image size.

File: README.md [hunk 2]
```diff
@@ -42,4 +42,21 @@ Now you can use:
 $ fs-go
 ```
 
-From anywhere in your system.
\ No newline at end of file
+From anywhere in your system.
+
+### BUILD AND RUN WITH DOCKER
+
+Build image
```
Comment (klipitkas) [inline README.md:49]:
```suggestion
To build a Docker image:
```

File: README.md [hunk 3]
```diff
@@ -42,4 +42,21 @@ Now you can use:
...
+```
+docker build -t fs:local .
+```
+
+Run it
```
Comment (klipitkas) [inline README.md:54]:
```suggestion
Run it using Docker:
```

File: README.md [hunk 4]
```diff
@@ -42,4 +42,21 @@ Now you can use:
...
+```
+docker run --rm -v $(pwd):/data -p 8080:8080 fs:local
+```
+
+Run in different port
```
Comment (klipitkas) [inline README.md:59]:
```suggestion
To run it using a different port:
```

File: README.md [hunk 5]
```diff
@@ -42,4 +42,21 @@ Now you can use:
...
+```
+docker run --rm -v $(pwd):/data -p 9000:9000 fs:local -addr :9000
```
Comment (klipitkas) [inline README.md:61]:
```suggestion
docker run --rm -v $(pwd):/data -p 9000:9000 fs:local -port 9000
```

**Review by klipitkas — CHANGES_REQUESTED** [review]

**Pull request merged**